_pages/how-we-work/equipment.md

```diff
@@ -17,6 +17,10 @@ GSA processes new badges every Wednesday. When your badge is ready, you’ll rec
 
 Badges are shipped to your local GSA office. You will receive an email once your badge has been shipped. At that point, you can make an appointment to pick it up. Your badge may be available during your orientation -- send an email to [hspd12.security@gsa.gov](mailto:hspd12.security@gsa.gov) to save yourself a trip. You may also ask your orientation leader to show you where to pick it up.
 
+## Business cards
+
+To make best use of our resources, we don't generally use business cards. If you need business cards for a specific meeting with partners, you can request [GSA business cards](https://requests.18f.gov/gsa18f/procurements/new), which are free and don't reflect TTS or 18F branding. Upon receiving a request, a member of Team Ops will log into Envision Business Cards, then reach out to you to retrieve the information you want on the card.
+To make best use of our resources, we don't generally use business cards. If you need business cards for a specific meeting with partners, though, you can request [GSA business cards](https://requests.18f.gov/gsa18f/procurements/new) [...]
```

Comment:
Ugh. I committed changes that duplicated this graf. Sorry bout that. I don't see how to correct it, either.

Reply:
@tbaxter-18f I believe I just fixed it in e7291d2. Please take a look when you get a chance. Thanks!